lib/govuk_personalisation/urls.rb

```diff
@@ -76,12 +76,23 @@ def self.find_govuk_url(var:, application:, path:)
   #
   # Otherwise, an application URL generated by Plek will be returned.
   #
-  # @param var         [String] the name of the variable to look up
-  # @param application [String] the name of the frontend application, passed to Plek (if the env var is set, this is ignored)
-  # @param path        [String] the path to use (if the env var is set this is ignored)
+  # @param var [String] the name of the variable to look up
+  # @param url [String] the url to default to if there is no environment variable set
   #
   # @return [String] the URL
   def self.find_external_url(var:, application:, path:)
     ENV.fetch("GOVUK_PERSONALISATION_#{var}_URI", "#{Plek.find(application)}#{path}")
```

Comment:
Do you also need to update the method to use `url`?

Reply:
Whoops